js/Msg.elm

```diff
@@ -18,6 +18,7 @@ type Msg
 
 type ReadyMsg
     = Ready
+    | NameChange String
```

Comment:
Calling this `NameInputChange` instead makes it more obvious it comes from the UI